testinfra/modules/package.py

```diff
@@ -165,7 +165,16 @@ def version(self):
 class RpmPackage(Package):
     @property
     def is_installed(self):
-        return self.run_test("rpm -q %s", self.name).rc == 0
+        result = self.run("rpm -q %s", self.name)
```

Comment:
You can keep using `run_test` so in case of exit code other than 0 or 1 this will raise.

Reply:
I cannot capture the `stdout` with `run_test`.
In the case exit code will be other than 1 or 0 it will throw anyways, as a `RuntimeError` exception.